BuildScript/BuildTargets/Testing.targets

```diff
@@ -257,7 +265,13 @@ $(_executionRuntime) $(_nunitCallArgumentString)
         InnerXml="$([MSBuild]::Unescape('%(_nunitDestinationFolderPath.SupportedRuntimeConfiguration)'))" />
   </Target>
 
-  <UsingTask AssemblyFile="C:\temp\Remotion.BuildScript.dll" TaskName="CreateTestConfigurations"/>
+  <PropertyGroup>
```

Comment:
This should be at the top of the file for readability. External-Import-like semantics at play here.